src/workspace.js

```diff
@@ -81,9 +85,14 @@ export default class WorkspaceClient {
    * Deletes a workspace.
    *
    * @param {String} name Name of the workspace to delete
-   * @param {String} recurse Flag to enable recursive deletion
+   * @param {Boolean} recurse Flag to enable recursive deletion
+   * 
+   * @returns {Boolean} If Deletion was successful
    */
   async delete (name, recurse) {
+    if (!recurse){
+      return false;
```

Comment:
Why are you exiting in case of `recurse=false`?

Reply:
my idea was to check if it is provided or not, I will remove it, since we do not do these checks in other function either